### DISCUSSION TITLE
Error on calling network.status(['stations']) with AP

### DISCUSSION THREAD

**Discussion by gcanzalo** [micropython/micropython#10159] [Libraries & Drivers]
Hi:

I have an AP on a D1 Mini running MicroPython v1.19.1 on 2022-06-18. Calling ap.status(['stations']) raises ```TypeError: can't convert 'list' object to str implicitly.```  I'm sure I'm doing something wrong, but don't know what.

Thanks for the help.

**Comment by glenn20** [comment]
This should work (pass string, not list):

```python
ap = network.WLAN(network.AP_IF)
ap.status('stations')
```

**Reply by gcanzalo** [reply]
Thanks for the reply. Tried that, it raises ```ValueError: unknown status param.```

**Reply by karfas** [reply]
In this case, you mistyped 'stations'.

**Reply by gcanzalo** [reply]
Directly from the REPL:

```
>>> ap.status('stations')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: unknown status param
>>>
```
